src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);net5.0;net5.0-windows</TargetFrameworks>
```

Comment:
Make sure that this doesn't use any ProjectReferences on `net5.0`.  It should build against the ref-pack in that case.